numpy/lib/_user_array_impl.py

```diff
@@ -0,0 +1,287 @@
+"""
+Standard container-class for easy multiple-inheritance.
+
+Try to inherit from the ndarray instead of using this class as this is not
+complete.
```

Comment:
I'd suggest replacing this docstring with the following to make it clearer it's not really meant to be used in new code:

```
Container class for backward compatibility with NumArray

The user_array.container class exists for backward compatibility with NumArray and is not meant to be used in new code. If you need to create an array container class, we recommend either creating a class that wraps an ndarray or subclasses ndarray.
```

Note to other reviewers, we chatted a bit about removing or deprecating this, but it has a few real usages on github and shows up in the docs with no warnings about not using it. We could deprecate it but there have been zero issues about it since NumPy moved to github so it's not a maintenance burden.

If anyone disagrees let us know and we can reconsider deprecating it or removing it.

Reply:
Sure! I updated the docstring.